ansible/playbooks/tasks/aws-cluster-hana.yaml

```diff
@@ -114,22 +180,28 @@
       ord_SAPHana
       Optional:
       cln_SAPHanaTpg_{{ sap_hana_install_sid }}_HDB{{ sap_hana_install_instance_number }}
-      msl_SAPHanaCtl_{{ sap_hana_install_sid }}_HDB{{ sap_hana_install_instance_number }}
+      {{ ms_saphanactl }}
   when: cluster_order | length == 0
 
-# Get current maintainence state
+# Get current maintenance state
 - name: Refresh cluster status
   ansible.builtin.command:
     cmd: crm configure show
   register: refreshed_crm_conf_show
   changed_when: false
 
-- name: Set crm facts
+- name: Refresh crm facts
   ansible.builtin.set_fact:
-    refreshed_crm_maintainence_mode: "{{ (refreshed_crm_conf_show.stdout | regex_search('maintenance-mode=([a-z]*)', '\\1'))[0] | default('unknown') }}"
+    refreshed_crm_maintenance_mode: "{{ (refreshed_crm_conf_show.stdout | regex_search('maintenance-mode=([a-z]*)', '\\1'))[0] | default('unknown') }}"
   changed_when: false
 
-- name: Ensure maintainence mode is disabled
+- name: Ensure maintenance mode is disabled
   ansible.builtin.command:
-    cmd: crm configure property maintenance-mode=false
-  when: refreshed_crm_maintainence_mode is true or refreshed_crm_maintainence_mode == 'unknown'
+    cmd: crm maintenance off
+  when: refreshed_crm_maintenance_mode is true or refreshed_crm_maintenance_mode == 'unknown'
+
+# For debug purpose only
```

Comment:
Do we like to keep it?

Reply:
... and after changes are done, we shoud start cluster back to normal

Reply:
Is it `name: Get cluster status at the end` a debug task?